packages/component-library/stories/BarChart.story.js

```diff
@@ -1,48 +1,135 @@
-      ]);
-      const dataKey = text("Data key", "ye");
-      const dataValue = text("Data values", "population");
-      const xLabel = text("xLabel", "Year");
-      const yLabel = text("yLabel", "Dogs");
-      const title = text("title", "Dogs with Money");
-      const subtitle = text(
-        "subtitle",
-        "Dogs in Portland with a net worth greater than $1,000"
-      );
+    .add(
+      "Standard",
+      () => {
+        const GROUP_IDS = {
+          LABELS: "Labels",
+          DATA: "Data"
+        };
+        const title = text("Title", "Dogs with Money", GROUP_IDS.LABELS);
+        const subtitle = text(
+          "Subtitle",
+          "Dogs in Portland with a net worth greater than $1,000",
+          GROUP_IDS.LABELS
+        );
+        const xLabel = text("X-axis label", "Year", GROUP_IDS.LABELS);
+        const yLabel = text("Y-axis label", "Dogs", GROUP_IDS.LABELS);
+        const dataKey = text("Data key", "ye", GROUP_IDS.DATA);
+        const dataValue = text("Data values", "population", GROUP_IDS.DATA);
+        const data = object(
+          "Data",
+          [
+            { ye: 1994, population: 2000 },
+            { ye: 1995, population: 8000 },
+            { ye: 1996, population: 6000 },
+            { ye: 1997, population: 3000 },
+            { ye: 1998, population: 1000 }
+          ],
+          GROUP_IDS.DATA
+        );
+        return (
+          <BarChart
+            data={data}
+            dataKey={dataKey}
+            dataValue={dataValue}
+            title={title}
+            subtitle={subtitle}
+            xLabel={xLabel}
+            yLabel={yLabel}
+          />
+        );
+      },
+      { notes }
+    )
+    .add(
+      "Custom",
+      () => {
+        const GROUP_IDS = {
+          LABELS: "Labels",
+          DATA: "Data",
+          CUSTOM: "Custom"
+        };
+        const title = text("Title", "Dogs with Money", GROUP_IDS.LABELS);
+        const subtitle = text(
+          "Subtitle",
+          "Dogs in Portland with a net worth greater than $1,000",
+          GROUP_IDS.LABELS
+        );
+        const xLabel = text("X-axis label", "Year", GROUP_IDS.LABELS);
+        const yLabel = text("Y-axis label", "Dogs", GROUP_IDS.LABELS);
+        const dataKey = text("Data key", "ye", GROUP_IDS.DATA);
+        const dataValue = text("Data values", "population", GROUP_IDS.DATA);
+        const data = object(
+          "Data",
+          [
+            { ye: 1994, population: 2000 },
+            { ye: 1995, population: 8000 },
+            { ye: 1996, population: 6000 },
+            { ye: 1997, population: 3000 },
+            { ye: 1998, population: 1000 }
+          ],
+          GROUP_IDS.DATA
+        );
+        const barWidth = number("Bar width", 37, {}, GROUP_IDS.CUSTOM);
+        const loading = boolean("Loading...", false, GROUP_IDS.CUSTOM);
+        const error = boolean("Error", false, GROUP_IDS.CUSTOM);
+        const getKeys = obj => {
```

Comment:
A little bit cleaner:
```
const getKeyNames = obj => {
  const keyNames = {};
  Object.keys(obj).forEach(key => {
    keyNames[key] = key;
  });
  return keyNames;
};
```

Reply:
Basically the same as before, just with cleaner naming

Reply:
Also, this function definition should be moved out of .add to the main file scope